markdown/clientaccess/kerberos.html.md.erb

```diff
@@ -515,6 +516,53 @@ Valid starting     Expires            Service principal
 
 After generating a ticket, you can connect to a HAWQ database as a kerberos-authenticated user using `psql` or other client programs.
 
+### <a id="change_ticket"></a>Changing the Ticket Renewal Interval
+
+The lifetime of the Kerberos ticket may need to be changed. The ticket lifetime is the minimum of the following values:
+
+* `max_life` in `kdc.conf` on the KDC servers.
```

Comment:
Not sure if we need to provide this level of detail. It might be better to just leave this as kerberos implementation-specific.

Reply:
Wen Lin wanted a section on the ticket lifetime and something like this.